### DISCUSSION TITLE
Tooltip delay

### DISCUSSION THREAD

**Discussion by blackjak231** [livewire/flux#565] [Feature Requests]
Hey there,

As a user, i would like to be able to delay the appearance of the tooltip with a simple "delay" attribute. For the "with tooltip trait", could simply be "tooltip-delay". A simple value in ms would work perfectly.

```html
<flux:tooltip content="Settings" delay="500">
    <flux:button icon="cog-6-tooth" icon-variant="outline" />
</flux:tooltip>

...

<flux:button tooltip="Settings" tooltip-delay="500" ... />
```

Cheers !

**Comment by joffreypersia** [comment]
UP ! Super important in UX !
